CHANGELOG.md

```diff
@@ -1,7 +1,14 @@
+## 0.4.7
```

Comment:
Yeah I think the release where the buffering went in should have been a Y, but it was accidentally done as a Z.